hepcrawl/pipelines.py

```diff
@@ -196,6 +196,9 @@ def open_spider(self, spider):
             ],
             CELERY_TASK_SERIALIZER='json',
             CELERY_RESULT_SERIALIZER='json',
+            CELERY_BROKER_TRANSPORT_OPTIONS=spider.settings['CELERY_BROKER_TRANSPORT_OPTIONS'],
+            CELERY_BROKER_CONNECTION_MAX_RETRIES=spider.settings['CELERY_BROKER_CONNECTION_MAX_RETRIES'],
```

Comment:
`CELERY_` prefix is wrong, causing the settings not to be used again, see https://docs.celeryproject.org/en/v4.4.7/userguide/configuration.html#new-lowercase-settings (you can also convert everything to the new lowercase variant). This works in invenio because the config loader has some logic to do the conversion.